sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -897,20 +897,20 @@ EXPLAIN FORMATTED
 struct<plan:string>
 -- !query output
 == Physical Plan ==
-Execute CreateViewCommand (1)
-   +- CreateViewCommand (2)
-         +- Project (5)
-            +- SubqueryAlias (4)
-               +- LogicalRelation (3)
+Execute CreateViewCommand (5)
```

Comment:
OK now I see what you mean. Let's avoid unnecessary behavior change and call `setOpId` before looking into the inner children.

Reply:
Ok, is it something that would be changed in a separate jira? It does seem weird that numbering for most nodes are post-order, and then when it's a node with inner children, it turns into pre-order